src/services/messengerSync/utils.js

```diff
@@ -1,9 +1,9 @@
 const { Configuration } = require('@schul-cloud/commons');
 const { userModel, displayName } = require('../user/model');
-const { schoolModel } = require('../school/model');
+const { schoolModel, SCHOOL_FEATURES } = require('../school/model');
 const roleModel = require('../role/model');
-const { courseModel } = require('../user-group/model');
-const { teamsModel } = require('../teams/model');
+const { courseModel, COURSE_FEATURES } = require('../user-group/model');
+const { teamsModel, TEAM_FEATURES } = require('../teams/model');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['TEAM_FEATURES' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5417363)